Description: Don't use rgb.txt from pstoimg
 This patch prevents pstoimg from using rgb.txt which can make latex2html calls
 fail.
Author: Atsuhito KOHDA <kohda@pm.tokushima-u.ac.jp>
Bug-Debian: http://bugs.debian.org/368693

--- latex2html-2008-debian1.orig/pstoimg.pin
+++ latex2html-2008-debian1/pstoimg.pin
@@ -1508,7 +1508,8 @@ sub crop_scale_etc {
       $pnmtoimg .= ' -interlace';
     }
     if($TRANSPARENT) {
-      $trans_color = $TRANSPARENT_COLOR||'gray85';
+      $trans_color = $TRANSPARENT_COLOR||'#d9d9d9'; # don't lookup rgb.txt
+      #$trans_color = $TRANSPARENT_COLOR||'gray85';
       $pnmtoimg .= ' -trans ' . L2hos->quote($trans_color);
     }
   }

